src/index.js

```diff
@@ -1,12 +1,6 @@
-import React from 'react';
```

Comment:
You need  to import `React` otherwise the `jsx` written inside this file will not be recognized

Reply:
Thank you very much. I fixed it.